docs/modules/deploy/pages/deploying-in-kubernetes.adoc

```diff
@@ -50,3 +59,401 @@ Since all the Operators mentioned above are generated from Helm Charts, their fu
 == Enterprise Operator (Preview)
 
 Hazelcast Enterprise Operator is currently in the early alpha preview stage, but it will soon become the preferred way of installing Hazelcast in Kubernetes/OpenShift environments. For details, please check xref:deploy:deploying-with-operator.adoc[].
+
+== Discovering Members Automatically
+
+To make it easier to set up clusters in Kubernetes, Hazelcast allows members to discover each other automatically, using discovery modes.
+
+=== Discovery Modes
+
+The following table summarizes the differences between the discovery modes: **Kubernetes API** and **DNS Lookup**.
+
+[cols="1a,1a,1a"]
+|===
+| | Kubernetes API  | DNS Lookup
+
+|Description    | Uses REST calls to Kubernetes master to fetch IP addresses of Pods | Uses DNS to resolve IPs of Pods related to the given service
+
+| Pros           | Flexible, supports **3 different options**:
+
+- Hazelcast cluster per service
+- Hazelcast cluster per multiple services (distinguished by labels)
+- Hazelcast cluster per namespace | **No additional configuration** required, resolving DNS does not require granting any permissions
+
+| Cons           | Requires setting up **RoleBinding** (to allow access to Kubernetes API)  | - Limited to **headless Cluster IP** service
+- Limited to **Hazelcast cluster per service**  |
+|===
+
+== Using Kubernetes in API Mode
+
+In **Kubernetes API** mode, each node makes a REST call to the Kubernetes master to discover the IP addresses of any Hazelcast members running in Pods.
+
+=== Granting Permissions to use Kubernetes API
+
+To use the Kubernetes API, you must grant certain permissions.
+
+To grant them for 'default' service account in 'default' namespace execute the following command.
+
+[source,shell]
+----
+kubectl apply -f https://raw.githubusercontent.com/hazelcast/hazelcast-kubernetes/master/rbac.yaml
```

Comment:
We need to change `https://raw.githubusercontent.com/hazelcast/hazelcast-kubernetes/master/rbac.yaml` to `https://raw.githubusercontent.com/hazelcast/hazelcast/master/kubernetes-rbac.yaml`. It's not only about this place, but everywhere in the Hazelcast documentation.